src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from 'react';
+import { Todo } from '../../types/Todo';
+import { Filter } from './components/Filter';
+
+interface Props {
+  todos: Todo[];
+  status: string;
+  handleClearCompleted: () => void;
+  setStatus: (value: string) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  setStatus,
+  handleClearCompleted,
+}) => {
+  const isActiveTodos = useMemo(
```

Comment:
```suggestion
  const activeTodos = useMemo(
```
It's an array of todos, not boolean value